controllers/imagelist/imagelist_controller.go

```diff
@@ -291,7 +291,15 @@ func (r *Reconciler) handleImageListEvent(ctx context.Context, imageList *eraser
 		pullSecrets = append(pullSecrets, corev1.LocalObjectReference{Name: secret})
 	}
 
+	labels := map[string]string{}
+	for k, v := range eraserConfig.Manager.AdditionalPodLabels {
+		labels[k] = v
+	}
+
 	jobTemplate := corev1.PodTemplateSpec{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: labels,
+		},
```

Comment:
same here, we don't need to set it here since its done in the imagejob_controller already

Reply:
I have removed it.